.travis.yml

```diff
@@ -1,13 +1,13 @@
 rvm:
-  - 2.1.9
-  - 2.2.5
-  - 2.3.1
+  - 2.1.10
+  - 2.2.4
+  - 2.3.3
+  - 2.4.0
 gemfile:
   - gemfiles/rails3.2.gemfile
-  - gemfiles/rails4.0.gemfile
-  - gemfiles/rails4.1.gemfile
```

Comment:
I don't get this. We stop testing against Rails 4.0 and 4.1, but we keep all the version checks in the code. How do we know it works if we don't run the tests?